src/screens/TopicMemberScreen/index.js

```diff
@@ -8,23 +8,23 @@ import ShareUtils from '../../utils/share';
 import dimen from '../../utils/dimen';
 import {getAllMemberTopic} from '../../service/topics';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
-import {colors} from '../../utils/colors';
 import StringConstant from '../../utils/string/StringConstant';
 import UsersFragment from '../DiscoveryScreenV2/fragment/UsersFragment';
 import {Context} from '../../context';
 import NavHeader from '../TopicPageScreen/elements/NavHeader';
+import {COLORS} from '../../utils/theme';
 
 const styles = StyleSheet.create({
   parentContainer: {
     flex: 1
   },
   fragmentContainer: {
     height: '100%',
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
   },
   fragmentContentContainer: {
     // flexGrow: 1,
-    backgroundColor: 'white'
+    backgroundColor: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hard-coded color value 'white' has been replaced with `COLORS.white`. This is a good practice as it centralizes the color definitions, making it easier to manage and update colors across the application.

```diff
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
-    backgroundColor: 'white'
+    backgroundColor: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->